pkg/kubelet/kubelet.go

```diff
@@ -1480,6 +1480,9 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 			if err := pcm.EnsureExists(pod); err != nil {
 				return fmt.Errorf("failed to ensure that the pod: %v cgroups exist and are correctly applied: %v", pod.UID, err)
 			}
+			if err := kl.containerManager.UpdateQOSCgroups(); err != nil {
```

Comment:
@sjenning -- this is the wrong order... this should be before EnsureExists, not after.  sorry for just catching this.